### PR TITLE
Add Nomad

### PR DESCRIPTION
**Pull request by SCKelemen**

### CODE REVIEW

**Comment by angrykoala** [comment]
Hi @SCKelemen Thanks for your PR.

I've been checking the docs available in esolangs.org on Nomad. I think it has some extremely interesting concepts and I'm happy to add it to the awesome-esloang list. However, I couldn't find examples nor proper documentation on how to actually code in it.

Awesome-esolang is a list intended to improve the visibility on esoteric languages to the general public or as humor. I would like to see some examples or basic guide (doesn't need to be comprehensive) of the language, so people unfamiliar with functional programming or lattice (like myself) can understand what makes this language unique

**Comment by SCKelemen** [comment]
I've added a few more examples, and will continue to improve the wiki page.
More detailed docs are in progress.

**Comment by angrykoala** [comment]
Hi @SCKelemen 
Sorry for taking so long to answer. The examples look great, thanks. :clap: 

I've been re-reading the docs, as per your description "Lightweight functional probabilistic programming language with a structural, lattice-based type system" I understand the main features that make Nomad unique are the lattice types and probabilistic programming.

* The lattice types is described in the wiki, although with few descriptions or references it is hard to follow for people unfamiliar with lattices like myself.
* The probabilistic programming aspect of the language is not really referenced in the docs and I don't see in the examples or any features regarding this (again, I have close to 0 knowledge on this so I may have missed something)

Remember that this list is intended for people interested in languages in general. I think it is better if the resources linked could cover **what** makes a language unique in a way people with little background on complex aspects of the language could understand.

**Comment by angrykoala** [comment]
Closed due to inactivity
